### ISSUE TITLE
[0.13.3] Could not parse compiler [...] - codesign disable flags missing

### ISSUE THREAD

**Issue by jessesquires** [realm/jazzy#1183]
Version `0.13.3` broke for my project: https://github.com/jessesquires/PresenterKit

Version `0.13.2` works. So this seems to be a regression.

### Script ([also here](https://github.com/jessesquires/PresenterKit/blob/develop/scripts/build_docs.sh))

```bash
PROJECT="PresenterKit"

jazzy \
    --clean \
    --author "Jesse Squires" \
    --author_url "https://jessesquires.com" \
    --github_url "https://github.com/jessesquires/$PROJECT" \
    --module "$PROJECT" \
    --source-directory . \
    --readme "README.md" \
    --documentation "Guides/*.md" \
    --output docs/
```

### Output using `0.13.3`

```bash
Running xcodebuild
Checking xcodebuild -showBuildSettings
Running xcodebuild
Could not parse compiler arguments from `xcodebuild` output.
Please confirm that `xcodebuild` is building a Swift module.
Saved `xcodebuild` log file: /var/folders/tx/s__fjnns3yd_23813_gwk7sh0000gn/T/xcodebuild-3AF8CAD1-53FA-4525-B8BF-C55AA351764D.log
Failed to generate documentation
Traceback (most recent call last):
	7: from /usr/local/var/rbenv/versions/2.6.5/bin/jazzy:23:in `<main>'
	6: from /usr/local/var/rbenv/versions/2.6.5/bin/jazzy:23:in `load'
	5: from /usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/bin/jazzy:15:in `<top (required)>'
	4: from /usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:76:in `build'
	3: from /usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:76:in `chdir'
	2: from /usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:78:in `block in build'
	1: from /usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/sourcekitten.rb:266:in `run_sourcekitten'
/usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/executable.rb:36:in `execute_command': /usr/local/var/rbenv/versions/2.6.5/lib/ruby/gems/2.6.0/gems/jazzy-0.13.3/bin/sourcekitten ["doc", "--module-name", "PresenterKit", "--"] (RuntimeError)

Running xcodebuild

Checking xcodebuild -showBuildSettings

Running xcodebuild

Could not parse compiler arguments from `xcodebuild` output.

Please confirm that `xcodebuild` is building a Swift module.

Saved `xcodebuild` log file: /var/folders/tx/s__fjnns3yd_23813_gwk7sh0000gn/T/xcodebuild-3AF8CAD1-53FA-4525-B8BF-C55AA351764D.log

Failed to generate documentation
```

**Comment by johnfairh** [comment]
If I clone that project and run `./scripts/build_docs.sh` it looks OK to me.  Xcode 11.4.1 / Ruby 2.5 or 2.6.4 :
```
; ./scripts/build_docs.sh 
Running xcodebuild
Parsing DismissButtonConfig.swift (1/5)
Parsing HalfModalPresentationController.swift (2/5)
Parsing PresentationType.swift (3/5)
Parsing UINavigationController+Extensions.swift (4/5)
Parsing UIViewController+Extensions.swift (5/5)
100% documentation coverage with 0 undocumented symbols
included 44 public or open symbols
skipped 21 private, fileprivate, or internal symbols (use `--min-acl` to specify a different minimum ACL)
building site
building search index
jam out ♪♫ to your fresh new docs in `docs`

    Warning: incorrect Jazzy installed! Please upgrade.
    Expected: 0.13.2
    Found: jazzy version: 0.13.3

    Download: https://github.com/realm/jazzy
    Install: gem install jazzy
```
Could you check the log file & see what xcodebuild is upset about?  The sourcekitten CLI it's
running that's mentioned in your crash report looks well-formed to me too.

**Comment by Yannick-91** [comment]
I have the same problem on my side.
When I looked at the logs :
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild -allowProvisioningUpdates

note: Using new build system
note: Planning build
note: Constructing build description
error: Communication with Apple failed: Your maximum App ID limit has been reached. You may create up to 10 App IDs every 7 days. (in target 'ChessGameTutorial' from project 'ChessGameTutorial')
error: No profiles for 'yannick.gablin.ChessGameTutorial' were found: Xcode couldn't find any iOS App Development provisioning profiles matching 'yannick.gablin.ChessGameTutorial'. (in target 'ChessGameTutorial' from project 'ChessGameTutorial')

** BUILD FAILED **

Any idea ? Because I spent my days trying to sort it out, and I am about to give up on Jazzy...

**Comment by johnfairh** [comment]
@Yannick-91 thanks for this - latest jazzy has a bug for projects that need code signing.

I don't know where the `-allowProvisioningUpdates` has come from: if you did not set it manually then there is a mystery.

As a workaround, try: `jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO`

**Comment by johnfairh** [comment]
@pjcau your project needs `jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO,-scheme,POC_C-MVVM-RX`

The CODE_SIGNING stuff is a temp workaround, but you will always need to pass `-scheme,...` because the default target doesn't build without its dependencies.

**Comment by Yannick-91** [comment]
Much better :-). Thanks for the suggestion. However (I know that I will sound like a dummy):

jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO
does not produce any content.

And 
jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO,-scheme,POC_C-MVVM-RX
does not go through.
Log is:
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO -scheme POC_C-MVVM-RX

Build settings from command line:
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO

xcodebuild: error: The project named "ChessGameTutorial" does not contain a scheme named "POC_C-MVVM-RX". The "-list" option can be used to find the names of the schemes in the project.

**Comment by johnfairh** [comment]
@Yannick-91 the POC_ stuff was addressed to another user.

Can you post the full output from running the jazzy command?  Not producing any content is a new one...

**Comment by Yannick-91** [comment]
Hi to the team...

**Command line and its output**

---------------------------------------------------
iMac-de-Yannick:ChessGameTutorial famille$ jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO,-scheme,POC_C-MVVM-RX
xcodebuild: error: The project named "ChessGameTutorial" does not contain a scheme named "POC_C-MVVM-RX". The "-list" option can be used to find the names of the schemes in the project.
Running xcodebuild
Could not successfully run `xcodebuild`.
Please check the build arguments.
Saved `xcodebuild` log file: /var/folders/5d/cxgj8jtd1hj8vb1yrt_04wsc0000gn/T/xcodebuild-978DADF8-D607-4136-BC5D-C006E3FE7AE3.log
Failed to generate documentation
Traceback (most recent call last):
	7: from /usr/local/bin/jazzy:23:in `<main>'
	6: from /usr/local/bin/jazzy:23:in `load'
	5: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/bin/jazzy:15:in `<top (required)>'
	4: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:76:in `build'
	3: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:76:in `chdir'
	2: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:78:in `block in build'
	1: from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/sourcekitten.rb:266:in `run_sourcekitten'
/Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/executable.rb:36:in `execute_command': /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/bin/sourcekitten ["doc", "--", "CODE_SIGN_IDENTITY\\=", "CODE_SIGNING_REQUIRED\\=NO", "CODE_SIGNING_ALLOWED\\=NO", "-scheme", "POC_C-MVVM-RX"] (RuntimeError)

xcodebuild: error: The project named "ChessGameTutorial" does not contain a scheme named "POC_C-MVVM-RX". The "-list" option can be used to find the names of the schemes in the project.

Running xcodebuild

Could not successfully run `xcodebuild`.

Please check the build arguments.

Saved `xcodebuild` log file: /var/folders/5d/cxgj8jtd1hj8vb1yrt_04wsc0000gn/T/xcodebuild-978DADF8-D607-4136-BC5D-C006E3FE7AE3.log

Failed to generate documentation
---------------------------------------------------

**And the associated log :**

---------------------------------------------------
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO -scheme POC_C-MVVM-RX

Build settings from command line:
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO

xcodebuild: error: The project named "ChessGameTutorial" does not contain a scheme named "POC_C-MVVM-RX". The "-list" option can be used to find the names of the schemes in the project.

**Comment by johnfairh** [comment]
@Yannick-91 the -scheme,POC_ stuff is not for your project.

We need the output of the command that you said is running OK but not producing any content, you wrote that this was `jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO`.

**Comment by Yannick-91** [comment]
Here it is:

iMac-de-Yannick:ChessGameTutorial famille$ jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO
Running xcodebuild
Parsing AppDelegate.swift (1/15)
Parsing Bishop.swift (2/15)
Parsing BoardIndex.swift (3/15)
Parsing ChessBoard.swift (4/15)
Parsing ChessGame.swift (5/15)
Parsing Dummy.swift (6/15)
Parsing ExtensionUIChessPiece.swift (7/15)
Parsing King.swift (8/15)
Parsing Knight.swift (9/15)
Parsing Pawn.swift (10/15)
Parsing Piece.swift (11/15)
Parsing Queen.swift (12/15)
Parsing Rook.swift (13/15)
Parsing SceneDelegate.swift (14/15)
Parsing ViewController.swift (15/15)
0% documentation coverage with 0 undocumented symbols
skipped 18 private, fileprivate, or internal symbols (use `--min-acl` to specify a different minimum ACL)
building site
building search index
jam out ♪♫ to your fresh new docs in `docs`

**Comment by johnfairh** [comment]
> skipped 18 private, fileprivate, or internal symbols (use --min-acl to specify a different minimum ACL)

Great, all is working fine.  You don't have any `public` declarations though.  Do `--min-acl private` to see everything.

**Comment by Yannick-91** [comment]
Working perfectly. Many thanks for your support :-)

**Comment by kmalyshev** [comment]
I know "could not parse compiler arguments from..." is a generic error. I can't get jazzy to work on an aggregated target. Target is in a separate project, added to a workspace.

Even if i delete all dependencies and it's an empty target I receive the following error:

`
Using config file /Users/user/project/.jazzy.yaml
Running xcodebuild
Checking xcodebuild -showBuildSettings
Running xcodebuild
Could not parse compiler arguments from `xcodebuild` output.
Please confirm that `xcodebuild` is building a Swift module.
Saved `xcodebuild` log file: /var/folders/34/87gnpk1d2ds42q69h2rn1g500000gp/T/xcodebuild-CC9F23A1-63D5-46B8-9F32-67794D85909F.log
Failed to generate documentation
bundler: failed to load command: jazzy (/usr/local/bin/jazzy)
RuntimeError: /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/bin/sourcekitten ["doc", "--module-name", "Root", "--", "build", "-workspace", "Workspace.xcworkspace", "-scheme", "All_Unit_Tests_macOS", "-UseModernBuildSystem\\=NO"]

Running xcodebuild

Checking xcodebuild -showBuildSettings

Running xcodebuild

Could not parse compiler arguments from `xcodebuild` output.

Please confirm that `xcodebuild` is building a Swift module.

Saved `xcodebuild` log file: /var/folders/34/87gnpk1d2ds42q69h2rn1g500000gp/T/xcodebuild-CC9F23A1-63D5-46B8-9F32-67794D85909F.log

Failed to generate documentation

  /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/lib/jazzy/executable.rb:36:in `execute_command'
  /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/lib/jazzy/sourcekitten.rb:266:in `run_sourcekitten'
  /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/lib/jazzy/doc_builder.rb:78:in `block in build'
  /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/lib/jazzy/doc_builder.rb:76:in `chdir'
  /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/lib/jazzy/doc_builder.rb:76:in `build'
  /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.1/bin/jazzy:15:in `<top (required)>'
  /usr/local/bin/jazzy:23:in `load'
  /usr/local/bin/jazzy:23:in `<top (required)>'
`

`
User defaults from command line:
    UseModernBuildSystem = NO

Build settings from command line:
    CODE_SIGN_IDENTITY = 
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO

Prepare build
note: Using legacy build system
=== BUILD AGGREGATE TARGET All_Unit_Tests_macOS OF PROJECT Root WITH CONFIGURATION Debug ===

Check dependencies

** BUILD SUCCEEDED **

Prepare build
note: Using legacy build system
=== CLEAN AGGREGATE TARGET All_Unit_Tests_macOS OF PROJECT Root WITH CONFIGURATION Debug ===

Check dependencies

Clean.Remove clean build/Intermediates.noindex/Root.build/Debug/All_Unit_Tests_macOS.build
    builtin-rm -rf /Users/user/project/build/Intermediates.noindex/Root.build/Debug/All_Unit_Tests_macOS.build

** CLEAN SUCCEEDED **

Prepare build
note: Using legacy build system
=== BUILD AGGREGATE TARGET All_Unit_Tests_macOS OF PROJECT Root WITH CONFIGURATION Debug ===

Check dependencies

** BUILD SUCCEEDED **
`

config: 
`
clean: true
module: Root
build_tool_arguments: [build,-workspace,Workspace.xcworkspace,-scheme,All_Unit_Tests_macOS,-UseModernBuildSystem=NO]
theme: fullwidth
`

Could give any hints on how to debug that further?

**Comment by johnfairh** [comment]
SourceKitten always adds in `build` or `clean build` so you don't need that.  I don't think this is causing the problem here but might confuse things down the line.

You need to figure out what `xcodebuild` command is needed to build the code of your module _Root_.  The `xcodebuild` output above says, I think, that it did `xcodebuild clean build -workspace Workspace.xcworkspace -scheme All_Unit_Tests_macOS -UseModernBuildSystem=NO`, and that there were no targets at all in that scheme, never mind one for the _Root_ module.

So find the scheme in Xcode that includes the Swift target _Root_ you want to document and build that scheme instead?  Or just the target, if it has no dependencies.

**Comment by jessesquires** [comment]
Thanks @johnfairh 💯 for your quick reply -- I apologize for my delayed response.

It appears that everything is working for me now (without any changes). I can't remember which Xcode version I was using when I opened this, but I'm on 11.4.1 now and it's working.

I guess Xcode was just being Xcode? 🤷‍♂️ 😅 

Sorry for the false alarm!

**Comment by johnfairh** [comment]
That's great,  thanks.  Keeping this issue to track the genuine codesign-related regression.

**Comment by pjcau** [comment]
I use this 

`jazzy --build-tool-arguments CODE_SIGN_IDENTITY=,CODE_SIGNING_REQUIRED=NO,CODE_SIGNING_ALLOWED=NO,-scheme,POC_C-MVVM-RX`

and I'll see this 



```
Showing All Messages
2020-05-14 21:41:39.768 xcodebuild[20480:3283079] [MT] PluginLoading: Required plug-in compatibility UUID C80A9C11-3902-4885-944E-A035869BA910 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/XXAlignOnSave.xcplugin' not present in DVTPlugInCompatibilityUUIDs

Running xcodebuild

Could not successfully run `xcodebuild`.

Please check the build arguments.

Saved `xcodebuild` log file: /var/folders/ct/x14jkxh92413l6kcqhswz3fw0000gn/T/xcodebuild-398D06F5-4C75-41FB-85B2-FAA746417000.log

Failed to generate documentation

/Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/executable.rb:36:in `execute_command': /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/bin/sourcekitten ["doc", "--", "CODE_SIGN_IDENTITY\\=", "CODE_SIGNING_REQUIRED\\=NO", "CODE_SIGNING_ALLOWED\\=NO", "-scheme", "POC_C-MVVM-RX"] (RuntimeError)

2020-05-14 21:41:39.768 xcodebuild[20480:3283079] [MT] PluginLoading: Required plug-in compatibility UUID C80A9C11-3902-4885-944E-A035869BA910 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/XXAlignOnSave.xcplugin' not present in DVTPlugInCompatibilityUUIDs

Running xcodebuild

Could not successfully run `xcodebuild`.

Please check the build arguments.

Saved `xcodebuild` log file: /var/folders/ct/x14jkxh92413l6kcqhswz3fw0000gn/T/xcodebuild-398D06F5-4C75-41FB-85B2-FAA746417000.log

Failed to generate documentation

	from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/sourcekitten.rb:266:in `run_sourcekitten'

	from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:78:in `block in build'

	from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:76:in `chdir'

	from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/lib/jazzy/doc_builder.rb:76:in `build'

	from /Library/Ruby/Gems/2.6.0/gems/jazzy-0.13.3/bin/jazzy:15:in `<top (required)>'

	from /usr/local/bin/jazzy:23:in `load'

	from /usr/local/bin/jazzy:23:in `<main>'

Command PhaseScriptExecution failed with a nonzero exit code

```


but if I see on logs

```
cat /var/folders/ct/x14jkxh92413l6kcqhswz3fw0000gn/T/xcodebuild-398D06F5-4C75-41FB-85B2-FAA746417000.log
2020-05-14 21:42:29.784 xcodebuild[20564:3284490] [MT] PluginLoading: Required plug-in compatibility UUID C80A9C11-3902-4885-944E-A035869BA910 for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/XXAlignOnSave.xcplugin' not present in DVTPlugInCompatibilityUUIDs
Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO -scheme POC_C-MVVM-RX

Build settings from command line:
    CODE_SIGN_IDENTITY =
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO
    TOOLCHAINS = com.apple.dt.toolchain.XcodeDefault

Resolve Package Graph

Resolved source packages:
  RxSwift: https://github.com/ReactiveX/RxSwift.git @ 5.1.1


note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description
Build system information
error: unable to attach DB: error: accessing build database "/Users/jonnycau/Library/Developer/Xcode/DerivedData/POC_C-MVVM-RX-ajxovvgcnhpfubbsavkboleldrgf/Build/Intermediates.noindex/XCBuildData/build.db": database is locked Possibly there are two concurrent builds running in the same filesystem location.

** BUILD FAILED **
```

**Comment by johnfairh** [comment]
> error: unable to attach DB: error: accessing build database
> "/Users/jonnycau/Library/Developer/Xcode/DerivedData/POC_C-MVVM-RX-
> ajxovvgcnhpfubbsavkboleldrgf/Build/Intermediates.noindex/XCBuildData/build.db":
> database is locked Possibly there are two concurrent builds running in the same filesystem
> location.

This error message is from `xcodebuild`; it's not one I've seen before. jazzy is not running more than one `xcodebuild` process.  I've personally not run jazzy from an Xcode script phase like your project is doing - might be that it can't work for this reason.

**Comment by pjcau** [comment]
> > error: unable to attach DB: error: accessing build database
> > "/Users/jonnycau/Library/Developer/Xcode/DerivedData/POC_C-MVVM-RX-
> > ajxovvgcnhpfubbsavkboleldrgf/Build/Intermediates.noindex/XCBuildData/build.db":
> > database is locked Possibly there are two concurrent builds running in the same filesystem
> > location.
> 
> This error message is from `xcodebuild`; it's not one I've seen before. jazzy is not running more than one `xcodebuild` process. I've personally not run jazzy from an Xcode script phase like your project is doing - might be that it can't work for this reason.

Yes, infact I think the issue is SPM and RXSwift. So I try to launch it from terminal after.

**Comment by pjcau** [comment]
> > > error: unable to attach DB: error: accessing build database
> > > "/Users/jonnycau/Library/Developer/Xcode/DerivedData/POC_C-MVVM-RX-
> > > ajxovvgcnhpfubbsavkboleldrgf/Build/Intermediates.noindex/XCBuildData/build.db":
> > > database is locked Possibly there are two concurrent builds running in the same filesystem
> > > location.
> > 
> > 
> > This error message is from `xcodebuild`; it's not one I've seen before. jazzy is not running more than one `xcodebuild` process. I've personally not run jazzy from an Xcode script phase like your project is doing - might be that it can't work for this reason.
> 
> Yes, infact I think the issue is SPM and RXSwift. So I try to launch it from terminal after.

I launched from terminal but It's the same and same issue

**Comment by pjcau** [comment]
I changed to use Carthage but I have the same issue 

```
cat  /var/folders/ct/x14jkxh92413l6kcqhswz3fw0000gn/T/xcodebuild-03BD226C-36ED-4B6D-9397-DF2527609446.log


Command line invocation:
    /Applications/Xcode.app/Contents/Developer/usr/bin/xcodebuild CODE_SIGN_IDENTITY= CODE_SIGNING_REQUIRED=NO CODE_SIGNING_ALLOWED=NO -scheme POC_C-MVVM-RX

Build settings from command line:
    CODE_SIGN_IDENTITY =
    CODE_SIGNING_ALLOWED = NO
    CODE_SIGNING_REQUIRED = NO
    TOOLCHAINS = com.apple.dt.toolchain.XcodeDefault

note: Using new build system
note: Building targets in parallel
note: Planning build
note: Constructing build description
Build system information
error: unable to attach DB: error: accessing build database "/Users/jonnycau/Library/Developer/Xcode/DerivedData/POC_C-MVVM-RX-ajxovvgcnhpfubbsavkboleldrgf/Build/Intermediates.noindex/XCBuildData/build.db": database is locked Possibly there are two concurrent builds running in the same filesystem location.

** BUILD FAILED **
```

**Comment by pjcau** [comment]
So, 

I tryed to call jazzy after the build, but not from xcode. In this way it works, but I cannot undestand because doesn't work inside the xcode

**Comment by johnfairh** [comment]
This was auto-closed early by GitHub magic, fix now in master via #1224.
